### DISCUSSION TITLE
LBFGS minimizer with JAX framework

### DISCUSSION THREAD

**Discussion by ZeshengLiu22** [jax-ml/jax#13558] [Q&A] [answered]
Does JAX have an LBFGS minimizer now? Checked with jax.scipy.minimize and it does not work for me. Optax seems not to have such an optimizer.

**Comment by adam-hartshorne** [comment, marked as answer]
You will find one here. It seems to sit above optax and provides a whole range of optimization functionality.

https://jaxopt.github.io/stable/index.html
